config/routes.rb

```diff
@@ -3,6 +3,23 @@
   root "users#index"
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
+  namespace :api do
+    namespace :v1 do
+      resources :posts, only: %i[index] do
+        resources :comments, only: %i[create index]
+      end
+     
+    end
+    
+  end
+
```

Comment:
@addod19 Sir, How can we add a comment to a post using get request?

Reply:
Sorry about my text error, the POST request is for adding comment to a post but when I mean by GET request is retrieving comments from a specific post. I hope this is clearer

Reply:
Hello @addod19 , thank you for your response. we have a get request to fetch a comment from a particular post. The end point  `api/v1/post/post_id/comments` can be used to GET comments for a particular post

Reply:
@addod19 Sir, We already have done what you are asking for us to make changes for.

Reply:
In your part of the code, kindly show me how to retrieve a comment from a specific post and from a specific user. For example; GET users/:user_id/posts/:id/comments

Reply:
Thank you for your reply once again @addod19. The route you requested for is in `api_v1_posts_comments_path` and this action is handled by `api/v1/comments#index` and this can be confirmed in the comments controller in the `api/v1/ `folder

Reply:
> Hello @addod19 , thank you for your response. we have a get request to fetch comments from a particular post. The end point `api/v1/posts/post_id/comments` can be used to GET comments for a particular post

Okay cool, I suspected this. But how can a user with id:2 retrieve it's comments from a specific post let say post_id: 3?
![Screenshot from 2022-03-05 00-07-02](https://user-images.githubusercontent.com/46388113/156858322-673ee508-bf9e-4ba9-8444-97efa2678d3b.png)
Kindly seed some data and try your API and if it works kindly submit another code review

Reply:
You can also use you API namespace like this ` /api/users/:user_id/posts or /api/v1/users/:user_id/posts` and you can add your post_id and comments at the end

Reply:
I don't know if am making sense. Otherwise request for a second opinion review